flake.nix

```diff
@@ -0,0 +1,121 @@
+{
+  description = "`gitoxide` compiled in a nix shell, using `crane` and `flakebox`.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-25.05";
+    flake-utils.url = "github:numtide/flake-utils";
+
+    fenix = {
+      url = "github:nix-community/fenix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
```

Comment:
I do not see this input used anywhere. It is not used as an argument in line 19, and I can't find any other changes that indicate it being used. What's going on?

```suggestion
```